src/components/molecules/card/CardHeader.tsx

```diff
@@ -31,6 +31,7 @@ const useStyles = makeStyles({
     position: 'absolute',
     width: 'fit-content',
     display: 'flex',
+    marginRight: '-24px',
```

Comment:
Using negative magin this way is a bad practice.
instead - since you already use `absolute` for positioning, add `right: 0`